src/app/screens/App/screens/Login/index.js

```diff
@@ -25,21 +28,33 @@ class LoginContainer extends Component {
     const email = this.state.email;
     const password = this.state.password;
 
-    const emailError = !EMAIL_REGEX.test(email);
-    const passwordError =
-      password.length < 8 ||
-      password.length > 52 ||
-      !HAS_LETTER_REGEX.test(password) ||
-      !HAS_NUMBER_REGEX.test(password);
-
-    if (emailError || passwordError) {
-      this.setState({ emailError, passwordError });
-    } else {
-      localStorage.setItem('currentUser', email);
-      this.props.history.push('/dashboard');
+    const invalidEmail = !EMAIL_REGEX.test(email);
+    const invalidPassword =
+      password.length < 8 || password.length > 52 || !HAS_NUMBER_REGEX.test(password);
+
+    this.setState({
+      emailError: invalidEmail && EMAIL_ERROR, //set either EMAIL_ERROR or false
+      passwordError: invalidPassword && PASSWORD_ERROR
+    });
+
+    if (!invalidEmail && !invalidPassword) {
+      this.requestAuthentication();
     }
   };
 
+  requestAuthentication = async () => {
+    const email = this.state.email;
+    const password = this.state.password;
+
+    axios
```

Comment:
when we configure axios we create a `api.js` folder into `config` folder.
We create an instances of Axios. https://github.com/axios/axios#creating-an-instance where configure it with the corresponding ABI_BASE_URL from the env file.